src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -20,10 +20,15 @@ export default class TranscriptArea {
   static #ENDPOINT_TRANSCRIPT = '/transcript';
   static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
   static #PARAM_ID = 'id';
-
+  static #TRANSCRIPT_ERROR_MESSAGE =
+      'Sorry, there is no transcript available for this lecture recording. :(';
+  // TODO: Update hasTranscript in the getter method for the transcript
+  // container once #286 is merged.
+  static #hasTranscript = true;
```

Comment:
Why is this field static? It is being set internally so I am not sure why it is static.